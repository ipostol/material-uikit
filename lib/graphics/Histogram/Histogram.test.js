'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Histogram = require('./Histogram');

var _Histogram2 = _interopRequireDefault(_Histogram);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Histogram component', function () {

  it('should render correctly', function () {

    var props = {
      histogram: [0, 100, 0, 50]
    };

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_Histogram2.default, props));

    expect(wrapper).toMatchSnapshot();
  });
});