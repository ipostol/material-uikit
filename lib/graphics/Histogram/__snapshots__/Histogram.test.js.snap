// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Histogram component should render correctly 1`] = `
<div
  className="histogram"
>
  <div
    className="bar"
    style={
      Object {
        "height": "0px",
      }
    }
  />
  <div
    className="bar"
    style={
      Object {
        "height": "34px",
      }
    }
  />
  <div
    className="bar"
    style={
      Object {
        "height": "0px",
      }
    }
  />
  <div
    className="bar"
    style={
      Object {
        "height": "17px",
      }
    }
  />
</div>
`;
